type_lens/type_view.py

```diff
@@ -137,3 +137,14 @@ def has_inner_subclass_of(self, cl: type[Any] | tuple[type[Any], ...]) -> bool:
             Whether any of the type's generic args are a subclass of the given type.
         """
         return any(t.is_subclass_of(cl) for t in self.inner_types)
+
+    def unwrap_optional(self) -> TypeView:
```

Comment:
What about `as_non_optional` or `ensure_non_optional`?

I suppose there's two things for the name to convey, if possible. 1. that it can be called on any type, not just optional unions, and 2. that it returns a different instance.

Reply:
`strip_optional`?

Reply:
I'm good with `strip_optional` :+1: